schemas/littlehorse/service.proto

```diff
@@ -252,6 +268,8 @@ service LittleHorse {
   rpc DeleteWfRun(DeleteWfRunRequest) returns (google.protobuf.Empty) {}
   // Deletes a TaskDef.
   rpc DeleteTaskDef(DeleteTaskDefRequest) returns (google.protobuf.Empty) {}
+  // Deletes a StructDef.
+  rpc DeleteStructDef(DeleteStructDefRequest) returns (google.protobuf.Empty) {}
```

Comment:
I'm wondering if we want to refactor all of these `DeleteFoo` rpc's to return the object that was deleted or throw `NOT_FOUND` otherwise.